test/test_sparql/test_datetime_processing.py

```diff
@@ -94,8 +94,8 @@ def test_datetime_duration_subs():
     SELECT (?d - ?duration AS ?next_year)
     WHERE {
         VALUES (?duration ?d) {
-            ("P1Y"^^xsd:yearMonthDuration"2019-05-28T12:14:45Z"^^xsd:dateTime)
-            ("P1Y"^^xsd:yearMonthDuration"2019-05-28"^^xsd:date)
+            ("P1Y"^^xsd:yearMonthDuration "2019-05-28T12:14:45Z"^^xsd:dateTime)
+            ("P1Y"^^xsd:yearMonthDuration "2019-05-28"^^xsd:date)
```

Comment:
This was an error I found in the datetime test function, there was a missing space between the literal values, causing incorrect tests to be run on durations.